misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -171,7 +171,8 @@ public List<String> skipList() {
                 "predeclared-module-prefix_02.bal", "object_type_def_source_44.bal", "record_type_def_source_27.bal",
                 "func_type_source_09.bal", "func_type_source_13.bal", "func_type_source_14.bal",
                 "func_type_source_15.bal", "func_type_source_16.bal", "import_decl_source_24.bal",
-                "member_access_expr_source_11.bal", "float_literal_source_08.bal");
+                "member_access_expr_source_11.bal", "float_literal_source_08.bal", "func_params_source_26.bal",
+                "func_params_source_27.bal", "func_params_source_28.bal", "func_params_source_29.bal");
```

Comment:
We need to run the formatter for the negative syntax as well. Shall we not skip these tests?